app/scripts/views/confirm.js

```diff
@@ -115,15 +121,22 @@ define(function (require, exports, module) {
               } else {
                 self.navigate('signup_complete');
               }
-            }, function (err) {
```

Comment:
Ahha, I'm reading the diff incorrectly. It was working just fine before, and we have the functional tests to prove it.